sig-documentation/README.md

```diff
@@ -0,0 +1,43 @@
+# Proposal: Spinnaker Documentation Special Interest Group
+
+## Roles & Responsibilities of the SIG Leads
+
+### SIG Lead(s):
+
+**[Brian Le](https://github.com/brian-armory)**  
+Technical Writer at Armory
+
+**[Ben Klein](https://github.com/fifthposition)**  
+Technical Writer at Pivotal
+
+TBD
+
+### SIG Lead Responsibility
+
+1. Coordinate communication.
+2. Set meeting dates and objectives for the coming year.
+3. Curate agenda for each meeting.
+4. Raise awareness and invite appropriate participants.
+5. Encourage and foster an inclusive environment.
+6. Facilitate the identification and development of resources (people or funding) in support of documentation for Spinnaker.
+
+
+### Meeting Cadence
+ - 1 hour every month on Google Hangouts
+ - TBD of every month at TBD PM PST
+
+# Responsibilities
+
+## Description
+
+Documentation SIG is responsible for the direction and organization of the Spinnaker documentation on https://www.spinnaker.io/. The SIG also acts as a place where users and developers can find collaborators and get help creating or modifying documentation.
```

Comment:
IMO: The documentation SIG should also be taking an active role in proposing and driving initiatives for code-driven doc generation. Take for example our swagger documentation. Take for example our public API or event stream.

Reply:
I think that's a good idea. @dorbin and @fifthposition, what are your thoughts?

Reply:
That sounds right to me.